Classes/ViewHelpers/GetDatabaseRecordViewHelper.php

```diff
@@ -83,8 +83,10 @@ public static function renderStatic(array $arguments, \Closure $renderChildrenCl
             ->where(
                 $queryBuilder->expr()->in('uid', $queryBuilder->createNamedParameter(explode($splitChar, $arguments['uidList']), Connection::PARAM_INT_ARRAY)),
             )
+            ->add('orderBy', 'FIELD(uid,' . $arguments['uidList'] . ')')
```

Comment:
for sql escaping i would prefer to
first, get an array of uids `$uids = GeneralUtility::intExplode(($splitChar, $arguments['uidList'], true);`
use this $uids for `$queryBuilder->expr()->in('uid', $queryBuilder->createNamedParameter($uids), Connection::PARAM_INT_ARRAY))`
use this $uids for order part: ` ->add('orderBy', 'FIELD(uid,' . implode(',', $uids] . ')')`
(this also unsure to have "," as delimiter for orderBy part, independent of `$spitChar`)